src/rgw/rgw_op.cc

```diff
@@ -715,6 +715,14 @@ int RGWGetObj::read_user_manifest_part(rgw_bucket& bucket, RGWObjEnt& ent, RGWAc
     return -EPERM;
   }
 
+  if (ent.size == 0) {
```

Comment:
"wיןבי can be easily created in DLO" is completely uninteresting as a comment (and can be confusing)